### DISCUSSION TITLE
2023-04-07 - Reflecting on April Fools - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by lukecookssw** [SSWConsulting/SSW.Rules.Content#5308] [Chewing the Fat]
cc: @adamcogan 

Discuss this week's CTF

**Comment by GordonBeeming** [comment]
![KermitSayWhatGIF](https://user-images.githubusercontent.com/5680199/230381071-917310dc-662d-4d43-8f87-0f569d3da4e1.gif)

For the record I think something like this would be super valuable to have, maybe I see that value more from RSA not being as culturally diverse in the workplace, and having been part of the mvp program for so long I see the value. Maybe it's not the same value in AUS and our other locations ... and although I called some of the april fools'ness last week... I must admit that I was also caught now that it was  (Probably from the value I believe it has... and my thoughts on it not being possible were shadowed by us wanting to push boundaries ) 

Sure, there would be some ironing out of the details around it but it could have massive value 🙂

**Comment by PennyWalker** [comment]
I love the idea. I think I think it’d be an awesome way to get know other cities and cultures with local guides and a great emersion. If we were able to work from another city, I’d definitely be keen and I’d love to host others from elsewhere! My spare room is always open for guests! :)

**Comment by rubycogan** [comment]
I am definitely pro helping all of SSW expand their world view. I believe that experiences like this are genuinely life changing and very rare and I think everyone should try to take this opportunity if they can.

**Comment by evecogan** [comment]
How about a SSW house swap 🏡

**Comment by calumjs** [comment]
I don't think you need any cultural exchange program. Unless you can somehow take the cost from my pre-tax income, or count it as a tax deduction, so it doesn't cost SSW more.

**Comment by drwharris** [comment]
The ability to travel and work is amazing and I think that if you can make it happen it would be rewarding for many.

**Comment by bahjatalaadelssw** [comment]
It doesn't need to be a SugarLearning item; however, because working remotely is so simple now if people want to move to different locations it can be an individual conversation.

**Comment by jimmidier** [comment]
Jokes are nice, but harmless ones are just better

**Comment by JerryLuo5799** [comment]
I think cultural exchange is great as long as it's not mandatory, even now that working remotely is convenient, but face-to-face work is a completely different experience.

**Comment by tino-liu** [comment]
The program is definitely a cool thing to have. Expanding worldview is a way to understand and embrace people and culture, that's how mutual respect are established. The program is definitely a great idea to bring people together.

**Comment by JaneYangssw** [comment]
Nice joke! And I suggest we can have more time to travel in the cultural exchange program, experiencing more cultural differences.

**Comment by Aibono1225** [comment]
Cultural exchange is a really cool idea. But we need to think about all the expenses that might come up during the travel.

**Comment by sylhuang** [comment]
I liked the idea of having a cultural exchange program. It's like another YOLO Day concept that would encourage me to do something normally not included in my daily plan.

**Comment by AttackOnMorty** [comment]
Looking forward to seeing the v2 of the Culture Exchange Program. It would be a wonderful journey.

**Comment by KristenHu** [comment]
I like this joke very much, I'm already looking forward to next year's April Fools' Day.

**Comment by johnxu16** [comment]
It was a little bit terrifying seeing a mandatory Culture Exchange Program.

As long as the program is optional, I would love to work and travel to another country.

I like the joke, especially the video part. It is not as official and educative as usual but really represents me different culture and how vivid life goes on in another country

**Comment by AnkyChe** [comment]
Culture Exchange Program is really nice as long as it's not mandatory.

**Comment by Berber-Mama** [comment]
As the person mainly responsible for reception in the China office, the only request is that you give me time to prepare in advance (at least 2 weeks) and come at the same time so that you can have the greatest experience in China!

**Comment by AshAnil** [comment]
I think a cultural exchange program is a cool idea to explore different countries. Moreover, cultural exchange programs can facilitate personal growth and development by encouraging individuals to step outside of their comfort zones.

**Comment by Freego1783** [comment]
I still think that this idea of cultural exchange program is nice !

**Comment by tiagov8** [comment]
At least we already have the bad example for the rule 😁

**Reply by leehawkinsssw** [reply]
Nicely played, @tiagov8

**Comment by leahy268** [comment]
As others said so long as it's not mandatory hell yes to Cultural exchange.  I really can't answer the expense part of it but it sounds great.

**Comment by leehawkinsssw** [comment]
Firstly, kudos for taking ownership of the problems caused by this April Fools.👏 I'm very happy to see this retro, although some kind of messaging from Adam shortly after the event would have been appropriate given the known upset 😢 this prank caused. 

Gary Klein's excellent idea of the "pre-mortem" would have gone some way to avoiding the problems caused in this case, I think. Having a diverse group🤼 review the prank beforehand would I'm sure have highlighted some of the issues before they were realised (diverse in terms of SSW tenure, cultural background, etc.)

I love the idea of a cultural exchange and I hope that this prank hasn't diluted the message that it's an awesome thing to offer SSW'ers. I look forward to seeing how the idea of this program benefits from the feedback in this CTF and maybe one day taking advantage of it myself.✈

**Comment by BrookJeynes** [comment]
I love the idea of an optional cultural exchange and am looking forward to it if it becomes a thing!

**Comment by tkapa** [comment]
Optional is good. I think Calum's idea is in the right area. The program doesn't really *need* to exist (officially) but people do need to know how SSW can facilitate us working from France, China, Melbourne etc for an extended period of time. Making it as cheap as possible is a win-win scenario.

**Comment by stephanfakossw** [comment]
The Cultural exchange is a great idea... We should be mindful to avoid any actions that might attract FBT.

**Comment by vladislav-kir** [comment]
Cultural exchange program would be awesome!

**Comment by danielmackay** [comment]
I like the idea of an optional culture exchange program.  Terms and Conditions TBC though. 🙂

I'm also fine with April fool jokes.  Personally, I didn't take this one too seriously.  It was definitely a great conversation starter around the coffee machine! 😆

**Comment by Marxoz** [comment]
I want to go to France please 🥐

I need to see whether there are more Jean's out there... 

So yes, I think the culture exchange program would be great. I find that attending retreats, conferences etc is a great way to meet, get to know, and built relationships with my fellow SSW employees. It would be great if we can do the same internationally.

**Comment by TobyChurches** [comment]
I'm definitely keen on the idea. Hopefully, it is available when I finish my degree at the end of this year 😁😁

**Comment by tomek-i** [comment]
I think it would be a great nice to have. I could actually visit my family :-)

**Comment by Ben0189** [comment]
Will be nice if we can have this program as optional

**Comment by cvionnet** [comment]
Apart from legal/logistical problems, I love the idea of a cultural exchange program.
This is something that few companies do and can add great value to SSW.

**Comment by amankumarrr** [comment]
Perhaps leaving it open as an option to work from any office without a time limit and with the expenses borne by the employees themselves would be better.

**Comment by AntPolkanov** [comment]
I previously worked at a company with a program similar to Cultural Exchange. We had a fixed budget for every employee which could be used once a year for travelling and working in any other office. The budget was enough to cover flights and accommodation for approximately 1 week. That was an amazing experience! ✈️

**Comment by Levijj22** [comment]
I still believe 🍄

**Comment by AlexRothwell** [comment]
While such a program doesn't appeal to me personally, I can absolutely see it as an amazing perk to offer and would be invaluable to many in the team

I really appreciate the effort taken for this retro. I'm keen to see what next year brings!

**Comment by christoment** [comment]
Cultural exchange is an interesting idea, especially for people who are still exploring things in their life 😄 
100% agree that this arrangement should be beneficial for both company and employees, so keen to see how we can dish this idea into a great one!

**Comment by camillars** [comment]
I loved the idea of a cultural exchange program. I hope this goes ahead so I can visit SSW France and China very soon :)

**Comment by PothieuG** [comment]
Totally ken on the idea ! I would be more than happy to show around Strasbourg and spend time with people who would like to visit our beautiful city !

**Comment by jimmy1992abc** [comment]
Program: ⭐⭐⭐⭐⭐
  - Compulsory: ⚠️⚠️⚠️⚠️⚠️
  - Optional and flexible: 💓💓💓💓💓

**Comment by SofieHong** [comment]
Looking forward to the April Fool CTF next year 🤪
Cultural Exchange Program: 🌟👍🏻💖

**Comment by chrisschultzssw** [comment]
Loved the video :)

**Comment by steven0x51** [comment]
The Culture Exchange is an exciting idea if it was optional. ❤

**Comment by JackDevAU** [comment]
Video 10/10 
Idea 10/10 
Mandatory -100/10

As others have noted, this would be a cool idea and should be something SSW should look into supporting!

**Comment by tombui99** [comment]
I only suspected that it was a joke (The "mandatory" part definitely gives it away). However, I actually liked the idea of a cultural exchange program since I love travelling and enjoy living in a different culture.

**Comment by matt-goldman** [comment]
I think this is a great idea. Whoever came up with this is a genius.

**Comment by SamBWagner** [comment]
It's a great idea and the April Fools joke was very funny. It just came at a time where the "Mandatory" extracurricular activities like the Azure Certs and what not were at an all time high. Then this was the straw that broke the camel's back. That's why I was rather annoyed and had my night and morning ruined with rage.

**Comment by Harry-Ross** [comment]
I thought it was a great idea as an optional program - would love to see it potentially developed in the future! Also in hindsight, the joke was bloody hillarious - would have been better though for some if they knew by the end of the day that it was a joke.

**Comment by ozdemir-mehmet** [comment]
Not much to add.

**Comment by jakebayliss** [comment]
Actually a cool idea if we can make it work out for everyone. When I go I'll take Mehmet with me

**Comment by zacharykeeping** [comment]
An optional exchange program is definitely a good idea!

**Comment by jernejk** [comment]
As an optional thing with more realistic benefits (we probably can't take care of entire families for the trip) would be great!
While my short-term focus will be more domestic, I would be open to eventually venture to China or France for a bit and experience the countries. 😊

It also helps that my family in both cases is ~2-4 hour flight away, making it a good opportunity to do a cultural exchange and visit family. 😊

Perhaps, 2 weeks in China/France, 2 weeks in Japan/Slovenia and then back 2 weeks in China/France would be an awesome trip with mid-stop to visit family and perhaps I could leave some of my bags in the office while I travel around, maximizing my trip. 😊 (day dreaming my next trip 😂)

**Comment by Hona** [comment]
The prospect of working overseas for a few months while making it financially more viable or even, assisted with help planning, or just knowing some coworkers will be there so you're not completely alone are some cool benefits.

I'm still not convinced this is an april fools!

**Comment by zl24622** [comment]
Culture exchange is an interesting idea. But it does need well preparation.
In the rules mentioned student language class, this might be a good option as the student visa is relatively long  than tourism visa and legal to work.

**Comment by william-liebenberg** [comment]
More people need to be "in on it" to not cause the chaos and loss of productivity that it did. However, the "optional" exchange idea is great and I can imagine quite a lot of people wanting to experience what its like to work and play in another country with a different culture.

**Comment by Abf09** [comment]
its good to have pranks often to have a good laugh and Cultural exchange is an amazing idea, If I am able to travel and manage work at the same time it'd be awesome.

**Comment by JackDuan123** [comment]
I would like to be in Eastern Europe

**Comment by JeanThirion** [comment]
Not funny.

**Comment by apforsyth** [comment]
Very funny and a brilliant video! But you'll never get me again ; )
